guvnor-ala/guvnor-ala-wildfly-provider/src/main/java/org/guvnor/ala/wildfly/access/WildflyAppState.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.guvnor.ala.wildfly.access;
+
+import java.util.Date;
+
+public class WildflyAppState {
```

Comment:
@manstis Initially, this was considered as part of the wildfly connector, so external to ALA, and we will need to transform from whatever the wildfly connector provides to the RuntimeState objects. I wanted to keep that code decoupled because we might need to extenralize this code later on to support multiple application servers (and version). At this point it might be not needed as you mention, but I would rather keep both types.